Flow.Launcher.Plugin/Result.cs

```diff
@@ -178,7 +178,7 @@ public override int GetHashCode()
         /// <inheritdoc />
         public override string ToString()
         {
-            return Title + SubTitle;
+            return Title + SubTitle + Score;
```

Comment:
what's the reason for this? I think you also need to update the hashcode if it is for replacing old result?

Reply:
It's not replacing any old result. I updated `ToString` to have an easier time debugging. Afaics, `ToString` was not used anywhere important.

I don't know how it is in .NET world , but in JVM world, `ToString` has nothing to do with `equals` and `hashCode`. I expect in .NET to be the same.

Reply:
Yeah it should have nothing to do with equals and hashcode, but I think a long time ago flow uses tostring for the usercount. Anyway I think it's fine.